source/opt/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_library(SpvToolsOpt
```

Comment:
Renamed.

Actually I'd think it would be nice to split `libSPIRV-Tools.so` into multiple smaller libraries via code reorganization (towards better structure). But that may break existing users and it's not a high priority task.